packages/dm-core-plugins/src/explorer/downloadNode.ts

```diff
@@ -0,0 +1,19 @@
+import { TreeNode } from '@development-framework/dm-core'
+import { AxiosResponse } from 'axios'
+
+export function downloadNode(node: TreeNode) {
+  node.tree.dmssApi
+    ._export({ pathAddress: node.getPath() }, { responseType: 'arraybuffer' })
+    .then(async (response: AxiosResponse) => {
+      const url = window.URL.createObjectURL(
+        new File([response.data], `${node.name}.zip`, {
+          type: 'application/zip',
+        })
+      )
+      const tempObjectElement = document.createElement('a')
```

Comment:
Also have a `createSyntethicFileDownload` helper function in dm-core-plugins/src/utils/fileUtils that does the same 🙂